common/test/test_utils.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import sys
+import io
+import tempfile
+import shutil
+import os
+from git import Repo
+sys.path.append('src')
+import utils
+
+
+# TODO: SetUp/TearDown to build tempdir with git project with shared_directory to test writing to
+#   each test gets its own subdirectory which is totally clean and unconnected
```

Comment:
Ungrammatical "...to test writing to each test gets..."

Reply:
Removed this entirely! I didn't even realize it wasn't removed previously.